### PR TITLE
hammer_cli_csv-2.0.0

### PR DESCRIPTION
**Pull request by thomasmckay**

### CODE REVIEW

File: rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 Summary: CSV input/output command plugin for the Hammer CLI
 Name:    %{?scl_prefix}rubygem-%{gem_name}
-Version: 1.0.1
+Version: 2.0.0
 Release: 8%{?dist}
```
Comment (ehelms) [inline rubygem-hammer_cli_csv/rubygem-hammer_cli_csv.spec]: Release should be set back to 1

**Comment by ehelms** [comment]
You added the binary and not the git annex for the package:

```
git annex add hammer_cli_csv-2.0.0.gem
```

**Comment by cpeters** [comment]
+1

**Comment by ehelms** [comment]
Thanks @thomasmckay !

**Pull request merged**